docker-compose.yml

```diff
@@ -2,16 +2,16 @@ version: '2'
 
 services:
   frontend:
+    image: index.docker.io/buildkite/frontend
     build: .
-    volumes:
-      - "./bundle-analysis:/frontend/bundle-analysis"
-      - "./dist:/frontend/dist"
-      - "./coverage:/frontend/coverage"
     environment:
-      BUILDKITE:
-      BUILDKITE_COMMIT:
-      BUILDKITE_ORGANIZATION_SLUG:
-      BUILDKITE_PIPELINE_SLUG:
-      CI:
-      EMOJI_HOST:
-      FRONTEND_HOST:
+      - EMOJI_HOST
+      - FRONTEND_HOST
+      - BUILDKITE
+      - BUILDKITE_COMMIT
+      - BUILDKITE_ORGANIZATION_SLUG
+      - BUILDKITE_PIPELINE_SLUG
+      - CI
+    volumes:
+      - ".:/frontend:cached"
```

Comment:
It's a very, very recent addition that allows docker for mac to make some tuning decisions on volume mounts from the host filesystem: https://docs.docker.com/docker-for-mac/osxfs-caching/

> cached: the host’s view is authoritative
> (permit delays before updates on the host appear in the container)

The TL;DR is that it makes anything that interacts with the filesystem much faster in the container at the expense of some eventual consistency with the host.